libibumad/umad.h

```diff
@@ -183,7 +183,7 @@ int umad_status(void *umad);
 ib_mad_addr_t *umad_get_mad_addr(void *umad);
 int umad_set_grh_net(void *umad, void *mad_addr);
 int umad_set_grh(void *umad, void *mad_addr);
-int umad_set_addr_net(void *umad, int dlid, int dqp, int sl, int qkey);
+int umad_set_addr_net(void *umad, __be16 dlid, __be32 dqp, int sl, __be32 qkey);
```

Comment:
Ok.

opensm uses this 3 times.  All 3 should be fine.

OPA management code does not use this and neither does the infiniband-diags...

So we are probably safe.